test/request-response-samples/RetrieveCustomerDetails__returning_a_date_should_parse_the_date_with_moment/soap.wsdl

```diff
@@ -0,0 +1,1494 @@
+<?xml version="1.0" encoding="UTF-8"?><wsdl:definitions xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/" name="CustomerManagement" targetNamespace="http://www.somecompany.co.uk/CustomerManagement" xmlns:bons0="http://www.somecompany.co.uk/RetrieveCustomerProductConfiguration" xmlns:bons2="http://www.somecompany.co.uk/CustomerManagement" xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap/" xmlns:soapenc="http://schemas.xmlsoap.org/soap/encoding/" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
Is it possible to shorten the `soap.wsdl`, `response.xml` and `response.json` files? It would help to keep the footprint of `soap` rather low :+1: